application/forms/Authentication/LoginForm.php

```diff
@@ -58,6 +63,14 @@ public function createElements(array $formData)
                 'class'         => isset($formData['username']) ? 'autofocus' : ''
             )
         );
+        $this->addElement(
+            'checkbox',
+            'rememberme',
+            [
+                'required'      => false,
```

Comment:
Please indent only up to the longest key + 1 space, i.e.

```php
$array = [
    'long-option' => 'long'
    'short'       => 'short'
];
```

PhpStorm should do that automatically for you. On mac, after selecting the code, press option + command + L.